tomcat9/src/main/java/com/linecorp/armeria/server/tomcat/TomcatService.java

```diff
@@ -516,6 +517,9 @@ private Request convertRequest(ServiceRequestContext ctx, AggregatedHttpRequest
         final HttpData content = req.content();
         coyoteReq.setInputBuffer((InputBuffer) INPUT_BUFFER_CONSTRUCTOR.invoke(content));
 
+        // Set the start time which is used by Tomcat access logging
+        coyoteReq.setStartTime(ctx.log().ensureAvailable(RequestLogProperty.REQUEST_START_TIME)
+                                  .requestStartTimeMillis());
```

Comment:
How about setting the request start time before returning a null value so that the invalid request is logged with the start time?